pkg/instance/ttlDestroyConsumer_test.go

```diff
@@ -17,17 +19,20 @@ import (
 func (s *ttlSuite) TestConsumeDeletesInstance() {
 	require := s.Require()
 
+	c := config.Config{UserService: config.Service{Username: "username", Password: "password"}}
+	uc := &userClient{}
+
 	consumer, err := rabbitmq.NewConsumer(
 		s.rabbitURI,
 		rabbitmq.WithConsumerPrefix("im-manager"),
 	)
 	require.NoError(err)
 	defer func() { require.NoError(consumer.Close()) }()
 
-	is := &instanceService{}
-	is.On("Delete", uint(1)).Return(nil)
+	is := &instanceService{userClient: uc}
```

Comment:
the instanceService does not depend on the userClient so you should not add it as a field.